pandas/core/strings.py

```diff
@@ -680,12 +680,33 @@ def str_repeat(arr, repeats):
 
     Parameters
     ----------
-    repeats : int or array
+    repeats : int or array of int
```

Comment:
I think array can give the impression we want a specific structure like numpy array. Same if we say list. I'd say we've been using sequence in cases like this.